alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/ui/ChannelDistributionUIConfig.java

```diff
@@ -79,9 +81,9 @@ public ChannelDistributionUIConfig(final String label, final String urlName, fin
 
     public abstract List<ConfigField> createChannelDistributionFields();
 
-    private Collection<String> getDescriptorNames(final DescriptorType descriptorType) {
+    private Collection<LabelValueSelectOption> getDescriptorNames(final DescriptorType descriptorType) {
         try {
-            return descriptorAccessor.getRegisteredDescriptorsByType(descriptorType).stream().map(RegisteredDescriptorModel::getName).collect(Collectors.toSet());
+            return descriptorAccessor.getRegisteredDescriptorsByType(descriptorType).stream().map(descriptor -> new LabelValueSelectOption(descriptor.getName())).collect(Collectors.toSet());
```

Comment:
We might be able to do something like this to avoid the lambda.
```java
return descriptorAccessor.getRegisteredDescriptorsByType(descriptorType)
        .stream()
        .map(RegisteredDescriptorModel::getName)
        .map(LabelValueSelectOption::new)
        .collect(Collectors.toSet());
```